src/client/src/rt-platforms/finsemble/index.ts

```diff
@@ -1,16 +1,33 @@
-import { BasePlatformAdapter } from '../platformAdapter'
-import { WindowConfig, AppConfig } from '../types'
+import { Platform } from '../platform'
+import { AppConfig, WindowConfig } from '../types'
 import { fromEventPattern } from 'rxjs'
+import DefaultRoute from '../defaultRoute'
+import Logo from '../logo'
+import { createDefaultPlatformWindow } from '../defaultPlatformWindow'
 
-export class Finsemble extends BasePlatformAdapter {
+export class Finsemble implements Platform {
   readonly name = 'finsemble'
   readonly type = 'desktop'
   readonly allowTearOff = true
+  style = {
+    height: '100%',
+  }
+  epics = []
+  PlatformHeader = () => null
+  PlatformControls = () => null
+  PlatformRoute = DefaultRoute
+  Logo = Logo
 
-  window = {
+  mainWindow = {
     close: () => window.close(),
+  }
 
-    open: (config: WindowConfig, onClose?: () => void) => Promise.resolve(window.open()),
```

Comment:
notice here that instead of inline `close` method, we are now wrapping web window into default `PlartfromWindow` with the help of `createDefaultPlatformWindow`. This is done so that later we can use enhanced platfrom window APIs (bring to front, etc)